src/main/kotlin/com/terraformation/backend/customer/db/UserStore.kt

```diff
@@ -216,6 +216,21 @@ class UserStore(
         .map { rowToIndividualUser(it) }
   }
 
+  /**
+   * Returns all users.
+   *
+   * @param requireOptIn Only return the users who are opted into email notifications.
+   */
+  fun fetchUsers(requireOptIn: Boolean = true): List<IndividualUser> {
+
+    return dslContext
+        .select(USERS.asterisk())
+        .from(USERS)
+        .apply { if (requireOptIn) where(USERS.EMAIL_NOTIFICATIONS_ENABLED.isTrue) }
```

Comment:
Missed this the first time around, but this needs a couple more conditions:

* `.where(USERS.USER_TYPE_ID.eq(UserType.Individual))` to filter out system and device manager users
* `.and(USERS.DELETED_TIME.isNull)` to filter out deleted users

Reply:
Thanks, that was why I was getting the system user!

Reply:
This will end up putting two `where`s on the query which will be a syntax error. Probably easiest to change this to an `and` and put it below the other conditions.